### ISSUE TITLE
Disconnect feature

### ISSUE THREAD

**Issue by frct1** [schroffl/teamspeak-query#19]
Hi @schroffl. I think in lib should be disconnect feature with full connection destroy.

**Comment by bigHapiness** [comment]
Through the test's (now buiding api with express and etc);
After app start: cpu usage is null
![image](https://user-images.githubusercontent.com/25178244/44618729-31d51e00-a884-11e8-991e-5a8bd4a4aa08.png)

After api call (connect to instance and return JSON output from hostinfo command) cpu usage growing with each api call
![image](https://user-images.githubusercontent.com/25178244/44618758-c475bd00-a884-11e8-827a-3fe8f074146a.png)


Api method source:
```
app.post('/status',async (req,res)=>{
    let credentials=req.body.instances;
    let statistics={};
    res.json(await getInfo(credentials));
});
function getInfo(credentials) {
        return new Promise(((resolve, reject) => {
            async.forEachOf(credentials,(instance,index,callback)=>{
                helper.connectAndAuth(instance).then(async connection=>{
                    let info=await connection.send('hostinfo').then(info=>{ return info; });
                    statistics[index]={status:'success',data:_.pick(info,'virtualservers_running_total','virtualservers_total_maxclients')};
                    callback();
                    connection.sock.destroy()
                }).catch(err=>{
                    let error=`${index}instance error`;
                    statistics[index]={status:"error",description:err};
                    callback(error);
                });
            },function (err) {
                if (err)console.log(err);
                resolve(statistics);
            })
        }))
    }
```

**Comment by schroffl** [comment]
I will take care of this next week.

**Comment by schroffl** [comment]
Published in [Version `3.1.0`](https://github.com/schroffl/teamspeak-query/releases/tag/v3.1.0)
